### DISCUSSION TITLE
Checking scope boxes when generating a personal access token

### DISCUSSION THREAD

**Discussion by pmdulaney** [community/community#38387] [New to GitHub] [answered]
There is some FOSS software for the Mac called OpenTerminal which I would like to install. When I attempted to get it via the command line my password was rejected (since Aug 13, 2021 passwords are no longer accepted). What boxes should I check? I assume that it wouldn't hurt to just check them all. Is that reasonable?

**Comment by airtower-luna** [comment, marked as answer]
> I assume that it wouldn't hurt to just check them all. Is that reasonable?

It's a security risk, because you'll likely grant much broader access than required.

But why would you need a token to download FOSS? That seems odd. A read-only clone of a public repository over HTTPS should work without one. :thinking:

**Reply by pmdulaney** [reply]
Thank you. I ended up giving the token very limited scope -- just write and read packages. But in the end when I tried the git clone command again I don't think it required a password or token. So I think you are right. But all's well that ends well. The app works like a charm.

**Reply by queenofcorgis** [reply]
Thanks for asking this question @pmdulaney and many thanks to the ever knowledge @airtower-luna for sharing her expertise 💃🏼
